test/tap/tests/unit/query_cache_unit-t.cpp

```
@@ -46,6 +47,21 @@ static uint64_t now_ms() {
 
+static unsigned char *dup_value(const char *text) {
+	return reinterpret_cast<unsigned char *>(strdup(text));
+}
+
+static uint32_t value_len(const char *text) {
+	return static_cast<uint32_t>(strlen(text) + 1);
+}
+
+static unsigned char *dup_fill(size_t len, char ch) {
+	unsigned char *buf = static_cast<unsigned char *>(malloc(len + 1));
+	memset(buf, ch, len);
```

Cppcheck (2.20.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)